src/Module/Install.php

```diff
@@ -61,45 +62,51 @@ public static function init()
 	public static function post()
 	{
 		$a = self::getApp();
+		$configCache = $a->getConfigCache();
 
 		switch (self::$currentWizardStep) {
 			case self::SYSTEM_CHECK:
+				self::checkSetting($configCache, $_POST, 'config', 'php_path');
+				break;
```

Comment:
Why did you add these two lines? They're the same as the next case, you can leave them combined like
```php
case self::SYSTEM_CHECK:
case self::DATABASE_CONFIG:
    self::checkSetting($configCache, $_POST, 'config', 'php_path');
    break;
```